### ISSUE TITLE
does it work with react-native?

### ISSUE THREAD

**Issue by sibelius** [DrVem/reicons#10]

**Comment by pedronauck** [comment]
I don't think so, because the generated template use a `<svg>` attributes, that doesn't work in React Native 😢  But I never tested...

**Comment by sibelius** [comment]
maybe it could work with 

https://github.com/react-native-community/react-native-svg

or

https://github.com/matc4/react-native-svg-uri
